File: LinkedList/LinkedListJava/Node.java
package LinkedList.LinkedListJava;

public class Node {
    
    private int dato;
    private Node prevNode;
    private Node nextNode;
    
    public Node(int dato) {
        this.dato = dato;
    }
    
    public int getDato() {
        return this.dato;
    }
    
    public void setDato(int dato) {
        this.dato = dato;
    }
    
    public Node getPrevNode() {
        return this.prevNode;
    }
    
    public void setPrevNode(Node prevNode) {
        this.prevNode = prevNode;
    }
    
    public Node getNextNode() {
        return this.nextNode;
    }
    
    public void setNextNode(Node nextNode) {
        this.nextNode = nextNode;
    }
    
}

File: stacks/Stacks-Python/Node.py

class Node:
    __siguiente = None

    def __init__(self, dato):
        self.__dato = dato

    def getDato(self) -> str:
        return self.__dato

    def setDato(self, dato):
        self.__dato = dato
    
    def getSiguiente(self):
        return self.__siguiente;

    def setSiguiente(self, siguiente):
        self.__siguiente = siguiente
